MdeModulePkg/Universal/Disk/DiskIoDxe/DiskIo.c

```diff
@@ -847,6 +847,7 @@ DiskIo2ReadWriteDisk (
   DISK_IO_SUBTASK         *Subtask;
   DISK_IO2_TASK           *Task;
   EFI_TPL                 OldTpl;
+  EFI_TPL                 OldTpl1;

```

Comment:
Recommend better names for both OldTpl local variables so the name indicates the context of raise/restore.